### PR TITLE
Add ToUInt64 to ShortChannelId 

### PR DESCRIPTION
**Pull request by joemphilips**

### CODE REVIEW

**Comment by knocte** [comment]
Just one nitpick, the net5 commit message explains what you're doing but not why you're doing it.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,7 +6,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, macos-latest, windows-latest]
-        dotnet: ['3.1.200']
+        dotnet: ['3.1.200', '5.0.8']
+        include:
+          - dotnet: '3.1.200'
+            framework: netcoreapp3.0
+          - dotnet: '3.0.8'
+            framework: net5
```
Comment (knocte) [inline .github/workflows/ci.yml:14]: also, is this right? why 3.0.8 maps to .NET5?

Comment (knocte) [reply .github/workflows/ci.yml:14]: ah that's a typo indeed, it should be 5.0.8

Comment (joemphilips) [reply .github/workflows/ci.yml:14]: Thanks, its typo

**Comment by knocte** [comment]
Is the net5 commit related to the ToUInt64 addition in any way?

**Comment by joemphilips** [comment]
> Is the net5 commit related to the ToUInt64 addition in any way?

Not really, I might split the PR.

**Comment by knocte** [comment]
Not really needed, was just curious. Your call :)

**Pull request merged**